services/hooks/src/main.js

```diff
@@ -129,7 +129,7 @@ const load = loader({
     requires: ['cfg', 'schemaset', 'Hook', 'LastFire', 'taskcreator', 'monitor', 'publisher', 'pulseClient'],
     setup: ({cfg, schemaset, Hook, LastFire, taskcreator, monitor, publisher, pulseClient}) => builder.build({
       rootUrl: cfg.taskcluster.rootUrl,
-      context: {Hook, LastFire, taskcreator, publisher},
+      context: {Hook, LastFire, taskcreator, publisher, denyList: cfg.pulse.denylist.exchanges},
```

Comment:
I think we'll introduce bugs by spelling it both `denylist` and `denyList`.  Let's just use the first (all-lowercase).